workbench-core/accounts/src/services/projectService.ts

```diff
@@ -127,91 +141,58 @@ export default class ProjectService {
       return { data: [project], paginationToken: undefined };
     }
 
-    // Else, member of more than 1 group, query for project items with pagination
-    // welcome to party town
```

Comment:
sad to see this comment go 😢

Reply:
me too, but the rewrite isn't quite worthy of party town status